### PR TITLE
Update si_resources.md

### PR DESCRIPTION
**Pull request by nadja-zlender**

### CODE REVIEW

**Review by Nazeeefa — CHANGES_REQUESTED** [review]
Thanks @nadja-zlender can you kindly check if any links are missing e.g. does ARIS have webpage? and if there are links to relevant guidelines/regulations/policies?

**Comment by nadja-zlender** [comment]
> Thanks @nadja-zlender can you kindly check if any links are missing e.g. does ARIS have webpage? and if there are links to relevant guidelines/regulations/policies?

We still need to check all of it within our group. Can we update the links later?

File: pages/national_resources/si_resources.md [hunk 1]
```diff
@@ -3,8 +3,10 @@
 title: Slovenia
 search_exclude: true
 country_code: SI
-contributors: [<!---REPLACE THIS with comma separated list of contributors--->]
-coordinators: [<!---REPLACE THIS with the name of data management coordinators of your ELIXIR node--->]
+page_id:  si_resources
+description: This page provides an overview of resources maintained by ELIXIR Slovenia for the life science research community in the country.
+contributors: [Brane Leskošek, Marko Vidak, Nadja Žlender]
+coordinators: [Brane Leskošek, Marko Vidak, Nadja Žlender]
```
Comment (bedroesb) [inline pages/national_resources/si_resources.md:9]: I am commenting on the wrong line, but if a few lines down you don't want to specify any specific training from Slovenia, you can delete that specific block.

**Comment by nadja-zlender** [comment]
> @nadja-zlender any updates? let us know if you need any help :)

I have updated the file.

**Comment by bedroesb** [comment]
@nadja-zlender Thanks a lot! Polonca Ferk was mentioned as coordinator but not as contributor. You can only be  coordinator if one is also a contributor (it puts the crown on the name). I swapped it for now, but please correct accordingly

**Review by bedroesb — APPROVED** [review]
Looks good on my side!

File: pages/national_resources/si_resources.md [hunk 2]
```diff
@@ -55,12 +44,27 @@ national_resources:
...
 
 ## Introduction 
-<!---General RDM considerations for your country, how to deal with RDM on a national level--->
+This page provides a general overview of national resources on Research Data Management (RDM) in Slovenia.
+An overview of data management services provided by ELIXIR Slovenia can be found on the [ELIXIR Slovenia website](https://elixir-slovenia.org/). 
 
 ## Funders
+The main public funder of research in Slovenia is the Slovenian Research and Innovation Agency (ARIS), which provides stable funding to the scientific community. The funding policy goals are to enable fair competition of scientists and to achieve excellence in science, reflected through participation of Slovenian scientists in European and other international projects. These projects result in high-impact publication with ARIS-funded scientists as leading authors. The Agency follows the policies of open science, scientific integrity, and gender equality.
+
+Some of ELIXIR Slovenia’s activities are funded by the European Union (EU), currently mainly through the ELIXIR-CONVERGE.
```
Comment (Nazeeefa) [inline pages/national_resources/si_resources.md:53]: The Converge has ended so this sentence can be revised perhaps?

**Review by Nazeeefa — CHANGES_REQUESTED** [review]

**Review by Nazeeefa — APPROVED** [review]

**Pull request merged**